src/main/java/sigma/task/DeadlineTask.java

```diff
@@ -0,0 +1,43 @@
+package sigma.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task.
+ */
+public class DeadlineTask extends Task {
+    private LocalDateTime date;
+
+    public DeadlineTask(String desc, LocalDateTime date) {
```

Comment:
I noticed you used a short form word "desc", may I suggest that you used the full word description to completely avoid ambiguity?
``` suggestion
    public DeadlineTask(String description, LocalDateTime date) {